csvcubed/csvcubed/cli/inspect/inspect_sparql_queries/select_single_unit_from_dsd.sparql

```diff
@@ -3,14 +3,14 @@ PREFIX csvw: <http://www.w3.org/ns/csvw#>
 
 SELECT DISTINCT ?unitUri ?unitLabel
 WHERE {   
-    ?column csvw:propertyUrl  "http://purl.org/linked-data/sdmx/2009/attribute#unitMeasure"^^csvw:uriTemplate;
+    ?column csvw:propertyUrl "http://purl.org/linked-data/sdmx/2009/attribute#unitMeasure"^^csvw:uriTemplate;
             csvw:valueUrl ?unitUriTemplate.
 
     BIND(str(?unitUriTemplate) as ?unitUri).
 
 
     OPTIONAL {
-        ?unit rdfs:titles ?unitLabel.
+        ?unit rdfs:label ?unitLabel.
```

Comment:
@GDonRanasinghe, do we have a unit test for this sparql query? If so we should be ensuring that we can match on the unitLabel where appropriate. I only noticed this after I started playing with some inputs.

Reply:
@robons I had a test for this but was commented out. I have enabled this test now and it assets for the uri and label.